Nach Wechsel auf 0.125.0 (be9e343a) Fehlermeldungen "Network Error. GET" im Frontend, jedoch nicht im Log

### Describe the bug

Seit dem Wechsel erhalte ich minütlich Meldungen des Typs jedoch ohne einen URL Anhang. 

"Network Error.
GET"

Schaue ich ins Debug Log werden keine GET Fehler angezeigt.

Eine Idee an welcher Stelle und wie ich der Meldung näher begegnen könnte?


### Steps to reproduce

1.
2.
3.
...


### Configuration details

```yaml
# open evcc at http://evcc.local:7070
network:
  schema: http
  host: evcc.local # .local suffix announces the hostname on MDNS
  port: 7080

log: debug
levels:
  cache: error
#  proxy-5200: trace
# unique installation id
plant: 37116dbc07c1e508a3730dbd3a94e1d603da226e639d9daebc1832d03118c510

interval: 30s # control cycle interval

sponsortoken: abcd

# sponsors can set telemetry: true to enable anonymous data aggregation
# see https://github.com/evcc-io/evcc/discussions/4554
telemetry: false

#modbusproxy:
#   - port: 5200
#     uri: 10.0.81.40:502
mqtt:
  clientid: evcc
  broker: localhost:1883
  topic: evcc

meters:
- type: template
  template: huawei-dongle-powersensor 
  id: 2  
  host: 10.0.81.40  
  port: 502  
  usage: grid  
  storageunit: 1  
  modbus: tcpip  
  name: grid1
- type: template
  template: huawei-dongle-powersensor 
  id: 2  
  host: 10.0.81.40  
  port: 502  
  usage: pv  
  storageunit: 1  
  modbus: tcpip  
  name: pv2
- type: template
  template: huawei-dongle-powersensor 
  id: 2  
  host: 10.0.81.40  
  port: 502  
  usage: battery  
  storageunit: 1  
  modbus: tcpip  
  name: battery3
- type: template
  template: huawei-dongle 
  id: 3  
  host: 10.0.81.40  
  port: 502  
  usage: pv  
  modbus: tcpip  
  name: pv5
- type: template
  template: huawei-dongle 
  id: 1  
  host: 10.0.81.40  
  port: 502  
  usage: pv  
  modbus: tcpip  
  name: pv6
- type: custom
  power:
   source: mqtt
   broker: localhost:1883
   topic: Balkonkraftwerk/power
   scale: 1000 
  name: pv7
chargers:
- type: template
  template: keba-modbus 
  modbus: tcpip
  id: 255
  host: 10.0.81.219  
  port: 502 
  name: wallbox7

loadpoints:
- title: Garage
  charger: wallbox7
  mode: pv # set default charge mode, use "off" to disable by default if charger is publicly available
  #resetOnDisconnect: true # set defaults when vehicle disconnects
  phases: 0 # electrical connection (normal charger: default 3 for 3 phase, 1p3p charger: 0 for "auto" or 1/3 for fixed phases)
  # remaining settings are experts-only and best left at default values
  priority: 0 # relative priority for concurrent charging in PV mode with multiple loadpoints (higher values have higher priority)
  soc:
    # polling defines usage of the vehicle APIs
    # Modifying the default settings it NOT recommended. It MAY deplete your vehicle's battery
    # or lead to vehicle manufacturer banning you from API use. USE AT YOUR OWN RISK.
    poll:
      # poll mode defines under which condition the vehicle API is called:
      #   charging: update vehicle ONLY when charging (this is the recommended default)
      #   connected: update vehicle when connected (not only charging), interval defines how often
      #   always: always update vehicle regardless of connection state, interval defines how often (only supported for single vehicle)
      mode: charging
        # poll interval defines how often the vehicle API may be polled if NOT charging
      interval: 60m
    estimate: true # set false to disable interpolating between api updates (not recommended)
  enable: # pv mode enable behavior
    delay: 5m # threshold must be exceeded for this long
    threshold: 0 # grid power threshold (in Watts, negative=export). If zero, export must exceed minimum charge power to enable
  disable: # pv mode disable behavior
    delay: 5m # threshold must be exceeded for this long
    threshold: 0 # maximum import power (W)
  guardDuration: 5m # switch charger contactor not more often than this (default 5m)
site:
  title: Mein Zuhause
  meters:
    grid: grid1
    pv:
    - pv2
    - pv5
    - pv6
    - pv7
#    battery:
#    - battery3
```


### Log details

```text
[lp-1 ] DEBUG 2024/04/24 14:46:07 charge power: 0W
[site ] DEBUG 2024/04/24 14:46:15 pv power: 7643W
[site ] DEBUG 2024/04/24 14:46:15 grid meter: -5285W
[site ] DEBUG 2024/04/24 14:46:18 grid currents: [-13.2 -9.81 3.24]A
[site ] DEBUG 2024/04/24 14:46:20 site power: -5285W
[lp-1 ] DEBUG 2024/04/24 14:46:20 charger status: A
[site ] DEBUG 2024/04/24 14:46:20 ----
[lp-1 ] DEBUG 2024/04/24 14:46:20 charge power: 0W
[site ] DEBUG 2024/04/24 14:47:03 pv power: 7889W
[site ] DEBUG 2024/04/24 14:47:06 grid meter: -5889W
[site ] DEBUG 2024/04/24 14:47:15 grid currents: [-14.5 -10.7 -3.25]A
[site ] DEBUG 2024/04/24 14:47:20 site power: -5889W
[lp-1 ] DEBUG 2024/04/24 14:47:20 charger status: A
[site ] DEBUG 2024/04/24 14:47:20 ----
[lp-1 ] DEBUG 2024/04/24 14:47:20 charge power: 0W
[site ] DEBUG 2024/04/24 14:47:26 pv power: 8490W
[site ] DEBUG 2024/04/24 14:47:28 grid meter: -6401W
[site ] DEBUG 2024/04/24 14:47:31 grid currents: [-15.4 -11.4 -3.38]A
[site ] DEBUG 2024/04/24 14:47:32 site power: -6401W
[lp-1 ] DEBUG 2024/04/24 14:47:32 charger status: A
[site ] DEBUG 2024/04/24 14:47:46 ----
[lp-1 ] DEBUG 2024/04/24 14:47:46 charge power: 0W
[site ] DEBUG 2024/04/24 14:48:15 pv power: 9948W
[site ] DEBUG 2024/04/24 14:48:20 grid meter: -7647W
[site ] DEBUG 2024/04/24 14:48:27 grid currents: [-17.6 -13.2 -4.14]A
[site ] DEBUG 2024/04/24 14:48:28 site power: -7647W
[lp-1 ] DEBUG 2024/04/24 14:48:28 charger status: A
[site ] DEBUG 2024/04/24 14:48:28 ----
[lp-1 ] DEBUG 2024/04/24 14:48:28 charge power: 0W
[site ] DEBUG 2024/04/24 14:48:34 pv power: 10120W
[site ] DEBUG 2024/04/24 14:48:35 grid meter: -8235W
[site ] DEBUG 2024/04/24 14:48:37 grid currents: [-18.8 -13.1 -4.53]A
[site ] DEBUG 2024/04/24 14:48:45 site power: -8235W
[lp-1 ] DEBUG 2024/04/24 14:48:45 charger status: A
[site ] DEBUG 2024/04/24 14:48:46 ----
[lp-1 ] DEBUG 2024/04/24 14:48:46 charge power: 0W
[site ] DEBUG 2024/04/24 14:49:29 pv power: 10941W
[site ] DEBUG 2024/04/24 14:49:33 grid meter: -8216W
[site ] DEBUG 2024/04/24 14:49:41 grid currents: [-18.3 -13.5 -4.04]A
[site ] DEBUG 2024/04/24 14:49:42 site power: -8216W
[lp-1 ] DEBUG 2024/04/24 14:49:42 charger status: A
[site ] DEBUG 2024/04/24 14:49:42 ----
[lp-1 ] DEBUG 2024/04/24 14:49:42 charge power: 0W
[site ] DEBUG 2024/04/24 14:49:53 pv power: 10442W
[site ] DEBUG 2024/04/24 14:49:54 grid meter: -7423W
[site ] DEBUG 2024/04/24 14:49:57 grid currents: [-16.9 -12.6 -3.83]A
[site ] DEBUG 2024/04/24 14:49:58 site power: -7423W
[lp-1 ] DEBUG 2024/04/24 14:49:58 charger status: A
[site ] DEBUG 2024/04/24 14:49:58 ----
[lp-1 ] DEBUG 2024/04/24 14:49:58 charge power: 0W
[site ] DEBUG 2024/04/24 14:50:27 pv power: 9975W
[site ] DEBUG 2024/04/24 14:50:32 grid meter: -6868W
[site ] DEBUG 2024/04/24 14:50:46 grid currents: [-16 -12 -3.57]A
[site ] DEBUG 2024/04/24 14:50:53 site power: -6868W
[lp-1 ] DEBUG 2024/04/24 14:50:53 charger status: A
[site ] DEBUG 2024/04/24 14:50:53 ----
[lp-1 ] DEBUG 2024/04/24 14:50:53 charge power: 0W
[site ] DEBUG 2024/04/24 14:51:08 pv power: 9598W
[site ] DEBUG 2024/04/24 14:51:09 grid meter: -7040W
[site ] DEBUG 2024/04/24 14:51:12 grid currents: [-16.2 -12.1 -3.6]A
[site ] DEBUG 2024/04/24 14:51:14 site power: -7040W
[lp-1 ] DEBUG 2024/04/24 14:51:14 charger status: A
[site ] DEBUG 2024/04/24 14:51:16 ----
[lp-1 ] DEBUG 2024/04/24 14:51:16 charge power: 0W
[site ] DEBUG 2024/04/24 14:51:41 pv power: 9495W
[site ] DEBUG 2024/04/24 14:51:47 grid meter: -7217W
[site ] DEBUG 2024/04/24 14:52:08 grid currents: [-16.3 -11.9 -3.5]A
[site ] DEBUG 2024/04/24 14:52:13 site power: -7217W
[lp-1 ] DEBUG 2024/04/24 14:52:13 charger status: A
[site ] DEBUG 2024/04/24 14:52:13 ----
[lp-1 ] DEBUG 2024/04/24 14:52:13 charge power: 0W
[site ] DEBUG 2024/04/24 14:52:25 pv power: 9262W
[site ] DEBUG 2024/04/24 14:52:27 grid meter: -6905W
[site ] DEBUG 2024/04/24 14:52:32 grid currents: [-16 -12 -3.49]A
[site ] DEBUG 2024/04/24 14:52:33 site power: -6905W
[lp-1 ] DEBUG 2024/04/24 14:52:33 charger status: A
[site ] DEBUG 2024/04/24 14:52:33 ----
[lp-1 ] DEBUG 2024/04/24 14:52:33 charge power: 0W
[site ] DEBUG 2024/04/24 14:52:59 pv power: 9103W
[site ] DEBUG 2024/04/24 14:53:04 grid meter: -6185W
[site ] DEBUG 2024/04/24 14:53:25 grid currents: [-14.9 -11.2 -3.35]A
[site ] DEBUG 2024/04/24 14:53:30 site power: -6185W
[lp-1 ] DEBUG 2024/04/24 14:53:30 charger status: A
[site ] DEBUG 2024/04/24 14:53:30 ----
[lp-1 ] DEBUG 2024/04/24 14:53:30 charge power: 0W
[site ] DEBUG 2024/04/24 14:53:43 pv power: 9013W
[site ] DEBUG 2024/04/24 14:53:49 grid meter: -5381W
[site ] DEBUG 2024/04/24 14:53:56 grid currents: [-17 -12.6 -3.86]A
[site ] DEBUG 2024/04/24 14:54:03 site power: -5381W
[lp-1 ] DEBUG 2024/04/24 14:54:03 charger status: A
[site ] DEBUG 2024/04/24 14:54:03 ----
[lp-1 ] DEBUG 2024/04/24 14:54:03 charge power: 0W
[site ] DEBUG 2024/04/24 14:54:33 pv power: 10990W
[site ] DEBUG 2024/04/24 14:54:37 grid meter: -17079W
[site ] DEBUG 2024/04/24 14:54:43 grid currents: [-35.1 -24.6 -13]A
[site ] DEBUG 2024/04/24 14:54:44 site power: -17079W
[lp-1 ] DEBUG 2024/04/24 14:54:44 charger status: A
[site ] DEBUG 2024/04/24 14:54:44 ----
[lp-1 ] DEBUG 2024/04/24 14:54:44 charge power: 0W
[site ] DEBUG 2024/04/24 14:54:53 pv power: 19305W
[site ] DEBUG 2024/04/24 14:54:54 grid meter: -17080W
[site ] DEBUG 2024/04/24 14:55:01 grid currents: [-35.1 -24.6 -13]A
[site ] DEBUG 2024/04/24 14:55:05 site power: -17080W
[lp-1 ] DEBUG 2024/04/24 14:55:05 charger status: A
[site ] DEBUG 2024/04/24 14:55:05 ----
[lp-1 ] DEBUG 2024/04/24 14:55:05 charge power: 0W
[site ] DEBUG 2024/04/24 14:55:49 pv power: 19503W
[site ] DEBUG 2024/04/24 14:55:53 grid meter: -17096W
[site ] DEBUG 2024/04/24 14:56:02 grid currents: [-35.2 -24.6 -12.9]A
[site ] DEBUG 2024/04/24 14:56:03 site power: -17096W
[lp-1 ] DEBUG 2024/04/24 14:56:03 charger status: A
[site ] DEBUG 2024/04/24 14:56:03 ----
[lp-1 ] DEBUG 2024/04/24 14:56:03 charge power: 0W
[site ] DEBUG 2024/04/24 14:56:19 pv power: 19499W
[site ] DEBUG 2024/04/24 14:56:20 grid meter: -17072W
[site ] DEBUG 2024/04/24 14:56:23 grid currents: [-35.1 -24.6 -13]A
[site ] DEBUG 2024/04/24 14:56:24 site power: -17072W
[lp-1 ] DEBUG 2024/04/24 14:56:24 charger status: A
[site ] DEBUG 2024/04/24 14:56:24 ----
[lp-1 ] DEBUG 2024/04/24 14:56:24 charge power: 0W
[site ] DEBUG 2024/04/24 14:56:52 pv power: 19527W
[site ] DEBUG 2024/04/24 14:57:02 grid meter: -12234W
[site ] DEBUG 2024/04/24 14:57:12 grid currents: [-24.4 -17.2 -6.99]A
[site ] DEBUG 2024/04/24 14:57:14 site power: -12234W
[lp-1 ] DEBUG 2024/04/24 14:57:14 charger status: A
[site ] DEBUG 2024/04/24 14:57:14 ----
[lp-1 ] DEBUG 2024/04/24 14:57:14 charge power: 0W
[site ] DEBUG 2024/04/24 14:57:33 pv power: 15793W
[site ] DEBUG 2024/04/24 14:57:34 grid meter: -15325W
[site ] DEBUG 2024/04/24 14:57:38 grid currents: [-30.7 -21.6 -10.5]A
[site ] DEBUG 2024/04/24 14:57:39 site power: -15325W
[lp-1 ] DEBUG 2024/04/24 14:57:39 charger status: A
[site ] DEBUG 2024/04/24 14:57:39 ----
[lp-1 ] DEBUG 2024/04/24 14:57:39 charge power: 0W
[site ] DEBUG 2024/04/24 14:58:00 pv power: 18397W
[site ] DEBUG 2024/04/24 14:58:03 grid meter: -14065W
[site ] DEBUG 2024/04/24 14:58:10 grid currents: [-28.2 -19.9 -9.31]A
[site ] DEBUG 2024/04/24 14:58:14 site power: -14065W
[lp-1 ] DEBUG 2024/04/24 14:58:14 charger status: A
[site ] DEBUG 2024/04/24 14:58:14 ----
[lp-1 ] DEBUG 2024/04/24 14:58:14 charge power: 0W
[site ] DEBUG 2024/04/24 14:58:25 pv power: 16754W
[site ] DEBUG 2024/04/24 14:58:26 grid meter: -10669W
[site ] DEBUG 2024/04/24 14:58:37 grid currents: [-23 -16.1 -6.73]A
[site ] DEBUG 2024/04/24 14:58:39 site power: -10669W
[lp-1 ] DEBUG 2024/04/24 14:58:39 charger status: A
[site ] DEBUG 2024/04/24 14:58:39 ----
[lp-1 ] DEBUG 2024/04/24 14:58:39 charge power: 0W
[site ] DEBUG 2024/04/24 14:59:24 pv power: 15165W
[site ] DEBUG 2024/04/24 14:59:27 grid meter: -13761W
[site ] DEBUG 2024/04/24 14:59:33 grid currents: [-26.6 -18.6 -8.04]A
[site ] DEBUG 2024/04/24 14:59:34 site power: -13761W
[lp-1 ] DEBUG 2024/04/24 14:59:34 charger status: A
[site ] DEBUG 2024/04/24 14:59:34 ----
[lp-1 ] DEBUG 2024/04/24 14:59:34 charge power: 0W
[site ] DEBUG 2024/04/24 14:59:53 pv power: 15371W
[site ] DEBUG 2024/04/24 14:59:54 grid meter: -9651W
[site ] DEBUG 2024/04/24 14:59:58 grid currents: [-20.9 -14.7 -4.9]A
[site ] DEBUG 2024/04/24 14:59:58 site power: -9651W
[lp-1 ] DEBUG 2024/04/24 14:59:58 charger status: A
[site ] DEBUG 2024/04/24 14:59:58 ----
[lp-1 ] DEBUG 2024/04/24 14:59:58 charge power: 0W
[site ] DEBUG 2024/04/24 15:00:23 pv power: 11631W
[site ] DEBUG 2024/04/24 15:00:28 grid meter: -6406W
[site ] DEBUG 2024/04/24 15:00:42 grid currents: [-15.8 -10.7 -3.17]A
[site ] DEBUG 2024/04/24 15:00:45 site power: -6406W
[lp-1 ] DEBUG 2024/04/24 15:00:45 charger status: A
[site ] DEBUG 2024/04/24 15:00:45 ----
[lp-1 ] DEBUG 2024/04/24 15:00:45 charge power: 0W
[site ] DEBUG 2024/04/24 15:01:08 pv power: 7894W
[site ] DEBUG 2024/04/24 15:01:09 grid meter: -4701W
[site ] DEBUG 2024/04/24 15:01:11 grid currents: [-12.5 -8.82 3.39]A
[site ] DEBUG 2024/04/24 15:01:12 site power: -4701W
[lp-1 ] DEBUG 2024/04/24 15:01:12 charger status: A
[site ] DEBUG 2024/04/24 15:01:12 ----
[lp-1 ] DEBUG 2024/04/24 15:01:12 charge power: 0W
[site ] DEBUG 2024/04/24 15:01:19 pv power: 7384W
[site ] DEBUG 2024/04/24 15:01:19 grid meter: -4728W
[site ] DEBUG 2024/04/24 15:01:37 grid currents: [-12.6 -8.95 3.37]A
[site ] DEBUG 2024/04/24 15:01:42 site power: -4728W
[lp-1 ] DEBUG 2024/04/24 15:01:42 charger status: A
[site ] DEBUG 2024/04/24 15:01:42 ----
[lp-1 ] DEBUG 2024/04/24 15:01:42 charge power: 0W
[site ] DEBUG 2024/04/24 15:02:11 pv power: 7681W
[site ] DEBUG 2024/04/24 15:02:16 grid meter: -5157W
[site ] DEBUG 2024/04/24 15:02:21 grid currents: [-13.4 -9.41 3.28]A
[site ] DEBUG 2024/04/24 15:02:22 site power: -5157W
[lp-1 ] DEBUG 2024/04/24 15:02:22 charger status: A
[site ] DEBUG 2024/04/24 15:02:22 ----
[lp-1 ] DEBUG 2024/04/24 15:02:22 charge power: 0W
[site ] DEBUG 2024/04/24 15:02:24 pv power: 7834W
[site ] DEBUG 2024/04/24 15:02:24 grid meter: -5116W
[site ] DEBUG 2024/04/24 15:02:26 grid currents: [-13.2 -9.31 3.3]A
[site ] DEBUG 2024/04/24 15:02:27 site power: -5116W
[lp-1 ] DEBUG 2024/04/24 15:02:27 charger status: A
[site ] DEBUG 2024/04/24 15:02:46 ----
[lp-1 ] DEBUG 2024/04/24 15:02:46 charge power: 0W
[site ] DEBUG 2024/04/24 15:02:55 pv power: 7782W
[site ] DEBUG 2024/04/24 15:02:56 grid meter: -5140W
[site ] DEBUG 2024/04/24 15:03:01 grid currents: [-13.3 -9.38 3.23]A
[site ] DEBUG 2024/04/24 15:03:02 site power: -5140W
[lp-1 ] DEBUG 2024/04/24 15:03:02 charger status: A
[site ] DEBUG 2024/04/24 15:03:16 ----
[lp-1 ] DEBUG 2024/04/24 15:03:16 charge power: 0W
[site ] DEBUG 2024/04/24 15:03:42 pv power: 8196W
[site ] DEBUG 2024/04/24 15:03:45 grid meter: -7769W
[site ] DEBUG 2024/04/24 15:03:51 grid currents: [-15.8 -11.1 -7.11]A
[site ] DEBUG 2024/04/24 15:03:52 site power: -7769W
[lp-1 ] DEBUG 2024/04/24 15:03:52 charger status: A
[site ] DEBUG 2024/04/24 15:03:52 ----
[lp-1 ] DEBUG 2024/04/24 15:03:52 charge power: 0W
[site ] DEBUG 2024/04/24 15:04:00 pv power: 9196W
[site ] DEBUG 2024/04/24 15:04:01 grid meter: -9292W
[site ] DEBUG 2024/04/24 15:04:04 grid currents: [-18.4 -12.9 -8.72]A
[site ] DEBUG 2024/04/24 15:04:05 site power: -9292W
[lp-1 ] DEBUG 2024/04/24 15:04:05 charger status: A
[site ] DEBUG 2024/04/24 15:04:16 ----
[lp-1 ] DEBUG 2024/04/24 15:04:16 charge power: 0W
[site ] DEBUG 2024/04/24 15:04:57 pv power: 13551W
[site ] DEBUG 2024/04/24 15:05:01 grid meter: -16423W
[site ] DEBUG 2024/04/24 15:05:14 grid currents: [-28.5 -17.6 -12.7]A
[site ] DEBUG 2024/04/24 15:05:17 site power: -16423W
[lp-1 ] DEBUG 2024/04/24 15:05:17 charger status: A
[site ] DEBUG 2024/04/24 15:05:17 ----
[lp-1 ] DEBUG 2024/04/24 15:05:17 charge power: 0W
[site ] DEBUG 2024/04/24 15:05:25 pv power: 13251W
[site ] DEBUG 2024/04/24 15:05:27 grid meter: -9502W
[site ] DEBUG 2024/04/24 15:05:30 grid currents: [-18.4 -13 -9.09]A
[site ] DEBUG 2024/04/24 15:05:37 site power: -9502W
[lp-1 ] DEBUG 2024/04/24 15:05:37 charger status: A
[site ] DEBUG 2024/04/24 15:05:46 ----
[lp-1 ] DEBUG 2024/04/24 15:05:46 charge power: 0W
[site ] DEBUG 2024/04/24 15:06:21 pv power: 8981W
[site ] DEBUG 2024/04/24 15:06:24 grid meter: -7216W
[site ] DEBUG 2024/04/24 15:06:28 grid currents: [-14.7 -10.5 -5.42]A
[site ] DEBUG 2024/04/24 15:06:30 site power: -7216W
[lp-1 ] DEBUG 2024/04/24 15:06:30 charger status: A
[site ] DEBUG 2024/04/24 15:06:30 ----
[lp-1 ] DEBUG 2024/04/24 15:06:30 charge power: 0W
[site ] DEBUG 2024/04/24 15:06:46 pv power: 8303W
[site ] DEBUG 2024/04/24 15:06:47 grid meter: -6040W
[site ] DEBUG 2024/04/24 15:06:57 grid currents: [-13.6 -9.72 -3.34]A
[site ] DEBUG 2024/04/24 15:07:06 site power: -6040W
[lp-1 ] DEBUG 2024/04/24 15:07:06 charger status: A
[site ] DEBUG 2024/04/24 15:07:06 ----
[lp-1 ] DEBUG 2024/04/24 15:07:06 charge power: 0W
[site ] DEBUG 2024/04/24 15:07:35 pv power: 7654W
[site ] DEBUG 2024/04/24 15:07:37 grid meter: -4808W
[site ] DEBUG 2024/04/24 15:07:43 grid currents: [-12.1 -8.69 3.09]A
[site ] DEBUG 2024/04/24 15:07:44 site power: -4808W
[lp-1 ] DEBUG 2024/04/24 15:07:44 charger status: A
[site ] DEBUG 2024/04/24 15:07:44 ----
[lp-1 ] DEBUG 2024/04/24 15:07:44 charge power: 0W
[site ] DEBUG 2024/04/24 15:07:53 pv power: 7041W
[site ] DEBUG 2024/04/24 15:07:56 grid meter: -4455W
[site ] DEBUG 2024/04/24 15:07:59 grid currents: [-11.7 -8.49 3.33]A
[site ] DEBUG 2024/04/24 15:08:00 site power: -4455W
[lp-1 ] DEBUG 2024/04/24 15:08:00 charger status: A
[site ] DEBUG 2024/04/24 15:08:00 ----
[lp-1 ] DEBUG 2024/04/24 15:08:00 charge power: 0W
[site ] DEBUG 2024/04/24 15:08:44 pv power: 6860W
[site ] DEBUG 2024/04/24 15:08:47 grid meter: -4419W
[site ] DEBUG 2024/04/24 15:08:55 grid currents: [-11.6 -8.43 3.33]A
[site ] DEBUG 2024/04/24 15:08:58 site power: -4419W
[lp-1 ] DEBUG 2024/04/24 15:08:58 charger status: A
[site ] DEBUG 2024/04/24 15:08:58 ----
[lp-1 ] DEBUG 2024/04/24 15:08:58 charge power: 0W
[site ] DEBUG 2024/04/24 15:09:10 pv power: 6743W
[site ] DEBUG 2024/04/24 15:09:11 grid meter: -4443W
[site ] DEBUG 2024/04/24 15:09:15 grid currents: [-11.7 -8.47 3.32]A
[site ] DEBUG 2024/04/24 15:09:15 site power: -4443W
[lp-1 ] DEBUG 2024/04/24 15:09:15 charger status: A
[site ] DEBUG 2024/04/24 15:09:16 ----
[lp-1 ] DEBUG 2024/04/24 15:09:16 charge power: 0W
[site ] DEBUG 2024/04/24 15:09:48 pv power: 6860W
[site ] DEBUG 2024/04/24 15:09:52 grid meter: -4620W
[site ] DEBUG 2024/04/24 15:10:02 grid currents: [-12.1 -8.7 3.26]A
[site ] DEBUG 2024/04/24 15:10:04 site power: -4620W
[lp-1 ] DEBUG 2024/04/24 15:10:04 charger status: A
[site ] DEBUG 2024/04/24 15:10:04 ----
[lp-1 ] DEBUG 2024/04/24 15:10:04 charge power: 0W
[site ] DEBUG 2024/04/24 15:10:20 pv power: 6938W
[site ] DEBUG 2024/04/24 15:10:24 grid meter: -4601W
[site ] DEBUG 2024/04/24 15:10:27 grid currents: [-12.1 -8.68 3.28]A
[site ] DEBUG 2024/04/24 15:10:28 site power: -4601W
[lp-1 ] DEBUG 2024/04/24 15:10:28 charger status: A
[site ] DEBUG 2024/04/24 15:10:28 ----
[lp-1 ] DEBUG 2024/04/24 15:10:28 charge power: 0W
[site ] DEBUG 2024/04/24 15:11:02 pv power: 6989W
[site ] DEBUG 2024/04/24 15:11:11 grid meter: -4735W
[site ] DEBUG 2024/04/24 15:11:20 grid currents: [-12.2 -8.69 3.28]A
[site ] DEBUG 2024/04/24 15:11:23 site power: -4735W
[lp-1 ] DEBUG 2024/04/24 15:11:23 charger status: A
[site ] DEBUG 2024/04/24 15:11:23 ----
[lp-1 ] DEBUG 2024/04/24 15:11:23 charge power: 0W
[site ] DEBUG 2024/04/24 15:11:32 pv power: 6977W
[site ] DEBUG 2024/04/24 15:11:33 grid meter: -4623W
[site ] DEBUG 2024/04/24 15:11:37 grid currents: [-12.1 -8.65 3.27]A
[site ] DEBUG 2024/04/24 15:11:51 site power: -4623W
[lp-1 ] DEBUG 2024/04/24 15:11:51 charger status: A
[site ] DEBUG 2024/04/24 15:11:51 ----
[lp-1 ] DEBUG 2024/04/24 15:11:51 charge power: 0W
[site ] DEBUG 2024/04/24 15:12:28 pv power: 6934W
[site ] DEBUG 2024/04/24 15:12:35 grid meter: -4881W
[site ] DEBUG 2024/04/24 15:12:54 grid currents: [-13.3 -9.38 -3.17]A
[site ] DEBUG 2024/04/24 15:13:02 site power: -4881W
[lp-1 ] DEBUG 2024/04/24 15:13:02 charger status: A
[site ] DEBUG 2024/04/24 15:13:02 ----
[lp-1 ] DEBUG 2024/04/24 15:13:02 charge power: 0W
[site ] DEBUG 2024/04/24 15:13:10 pv power: 9027W
[site ] DEBUG 2024/04/24 15:13:11 grid meter: -6877W
[site ] DEBUG 2024/04/24 15:13:14 grid currents: [-16.5 -11.8 -3.79]A
[site ] DEBUG 2024/04/24 15:13:16 site power: -6877W
[lp-1 ] DEBUG 2024/04/24 15:13:16 charger status: A
[site ] DEBUG 2024/04/24 15:13:16 ----
[lp-1 ] DEBUG 2024/04/24 15:13:16 charge power: 0W
[site ] DEBUG 2024/04/24 15:13:46 pv power: 9044W
[site ] DEBUG 2024/04/24 15:13:49 grid meter: -8222W
[site ] DEBUG 2024/04/24 15:14:00 grid currents: [-26.9 -17.7 -7.9]A
[site ] DEBUG 2024/04/24 15:14:03 site power: -8222W
[lp-1 ] DEBUG 2024/04/24 15:14:03 charger status: A
[site ] DEBUG 2024/04/24 15:14:03 ----
[lp-1 ] DEBUG 2024/04/24 15:14:03 charge power: 0W
[site ] DEBUG 2024/04/24 15:14:21 pv power: 18177W
[site ] DEBUG 2024/04/24 15:14:22 grid meter: -17050W
[site ] DEBUG 2024/04/24 15:14:26 grid currents: [-34.9 -23.4 -12.4]A
[site ] DEBUG 2024/04/24 15:14:27 site power: -17050W
[lp-1 ] DEBUG 2024/04/24 15:14:27 charger status: A
[site ] DEBUG 2024/04/24 15:14:27 ----
[lp-1 ] DEBUG 2024/04/24 15:14:27 charge power: 0W
[site ] DEBUG 2024/04/24 15:14:53 pv power: 19083W
```


### What type of operating system are you running?

Linux

### Version

evcc version 0.125.0 (be9e343a)

> Zeigt die Browser Console Fehler an?

tatsächlich! 

Scheinbar wird zum einen wegen der definierten battery3 etwas geworfen, ich war davon ausgegangen, dass es reicht die Verwendung unter site: auszukommentieren (siehe evcc.yaml).

Sollte ich die Definition besser in Gänze entfernen oder auskommentieren?

Die anderen Huawei Geräte meckert er auch an, hat sich an der Huawei Abfrage etwas geändert oder am logging? Den Intervall für den Dongle habe ich nicht verringert. (vorher Version 0.124.1)

`Config-CieCu7-0.js:1 Error fetching device values for meter battery3 
ke {message: 'Network Error', name: 'AxiosError', code: 'ERR_NETWORK', config: {…}, request: XMLHttpRequest, …}
index-CI2EAdOo.js:23 
 GET https://lademanagement.eldrik.de/api/config/devices/meter/battery3/status net::ERR_HTTP2_SERVER_REFUSED_STREAM
index-CI2EAdOo.js:87 no data received, assume we are offline
Config-CieCu7-0.js:1 Error fetching device values for meter grid1 
ke {message: 'Network Error', name: 'AxiosError', code: 'ERR_NETWORK', config: {…}, request: XMLHttpRequest, …}
index-CI2EAdOo.js:23 
 GET https://lademanagement.eldrik.de/api/config/devices/meter/grid1/status net::ERR_HTTP2_SERVER_REFUSED_STREAM
index-CI2EAdOo.js:87 no data received, assume we are offline
Config-CieCu7-0.js:1 Error fetching device values for meter pv5 
ke {message: 'Network Error', name: 'AxiosError', code: 'ERR_NETWORK', config: {…}, request: XMLHttpRequest, …}
index-CI2EAdOo.js:23 
 GET https://lademanagement.eldrik.de/api/config/devices/meter/pv5/status net::ERR_HTTP2_SERVER_REFUSED_STREAM
Config-CieCu7-0.js:1 Error fetching device values for meter pv2 
ke {message: 'Network Error', name: 'AxiosError', code: 'ERR_NETWORK', config: {…}, request: XMLHttpRequest, …}
index-CI2EAdOo.js:23 
 GET https://lademanagement.eldrik.de/api/config/devices/meter/pv2/status net::ERR_HTTP2_SERVER_REFUSED_STREAM
Config-CieCu7-0.js:1 Error fetching device values for meter pv6 
ke {message: 'Network Error', name: 'AxiosError', code: 'ERR_NETWORK', config: {…}, request: XMLHttpRequest, …}
index-CI2EAdOo.js:23 
 GET https://lademanagement.eldrik.de/api/config/devices/meter/pv6/status net::ERR_HTTP2_SERVER_REFUSED_STREAM
index-CI2EAdOo.js:87 no data received, assume we are offline
Config-CieCu7-0.js:1 Error fetching device values for meter battery3 
ke {message: 'Network Error', name: 'AxiosError', code: 'ERR_NETWORK', config: {…}, request: XMLHttpRequest, …}
index-CI2EAdOo.js:23 
 GET https://lademanagement.eldrik.de/api/config/devices/meter/battery3/status net::ERR_HTTP2_SERVER_REFUSED_STREAM`

--- a/assets/js/app.js
+++ b/assets/js/app.js
@@ -28,6 +28,7 @@ const app = createApp({
   },
   methods: {
     raise: function (msg) {
+      if (this.offline) return;
       if (!msg.level) msg.level = "error";
       const now = new Date();
       const latestMsg = this.notifications[0];

--- a/assets/js/auth.js
+++ b/assets/js/auth.js
@@ -4,14 +4,14 @@ import Modal from "bootstrap/js/dist/modal";
 
 const auth = reactive({
   configured: true,
-  loggedIn: false,
+  loggedIn: null, // true / false / null (unknown)
   nextUrl: null,
 });
 
 export async function updateAuthStatus() {
   try {
     const res = await api.get("/auth/status", {
-      validateStatus: (code) => [200, 501].includes(code),
+      validateStatus: (code) => [200, 501, 500].includes(code),
     });
     if (res.status === 501) {
       auth.configured = false;
@@ -19,6 +19,10 @@ export async function updateAuthStatus() {
     if (res.status === 200) {
       auth.configured = true;
       auth.loggedIn = res.data === true;
+    }
+    if (res.status === 500) {
+      auth.loggedIn = null;
+      console.log("unable to fetch auth status", res);
     }
   } catch (e) {
     console.log("unable to fetch auth status", e);
@@ -36,7 +40,11 @@ export async function logout() {
 }
 
 export function isLoggedIn() {
-  return auth.loggedIn;
+  return auth.loggedIn === true;
+}
+
+export function statusUnknown() {
+  return auth.loggedIn === null;
 }
 
 export function isConfigured() {

--- a/assets/js/components/TelemetrySettings.vue
+++ b/assets/js/components/TelemetrySettings.vue
@@ -75,9 +75,12 @@ export default {
 				return;
 			}
 			try {
-				const response = await api.get("settings/telemetry");
-				console.log("update in settings", response.data.result);
-				settings.telemetry = response.data.result;
+				const response = await api.get("settings/telemetry", {
+					validateStatus: () => true,
+				});
+				if (response.status === 200) {
+					settings.telemetry = response.data.result;
+				}
 			} catch (err) {
 				console.error(err);
 			}

--- a/assets/js/router.js
+++ b/assets/js/router.js
@@ -2,7 +2,7 @@ import { createRouter, createWebHashHistory } from "vue-router";
 import Modal from "bootstrap/js/dist/modal";
 import Main from "./views/Main.vue";
 import { ensureCurrentLocaleMessages } from "./i18n";
-import { openLoginModal, updateAuthStatus, isLoggedIn } from "./auth";
+import { openLoginModal, statusUnknown, updateAuthStatus, isLoggedIn } from "./auth";
 
 function hideAllModals() {
   [...document.querySelectorAll(".modal.show")].forEach((modal) => {
@@ -18,7 +18,7 @@ function hideAllModals() {
 
 async function ensureAuth(to) {
   await updateAuthStatus();
-  if (!isLoggedIn()) {
+  if (!isLoggedIn() && !statusUnknown()) {
     openLoginModal(to.path);
     return false;
   }

--- a/assets/js/views/Config.vue
+++ b/assets/js/views/Config.vue
@@ -466,14 +466,16 @@ export default {
 		},
 		async updateValues() {
 			clearTimeout(this.deviceValueTimeout);
+			if (!this.offline) {
+				const promises = [
+					...this.meters.map((meter) => this.updateDeviceValue("meter", meter.name)),
+					...this.vehicles.map((vehicle) =>
+						this.updateDeviceValue("vehicle", vehicle.name)
+					),
+				];
 
-			const promises = [
-				...this.meters.map((meter) => this.updateDeviceValue("meter", meter.name)),
-				...this.vehicles.map((vehicle) => this.updateDeviceValue("vehicle", vehicle.name)),
-			];
-
-			await Promise.all(promises);
-
+				await Promise.all(promises);
+			}
 			this.deviceValueTimeout = setTimeout(this.updateValues, 10000);
 		},
 		deviceTags(type, id) {

